gradle/root/testing.gradle

```diff
@@ -42,6 +42,7 @@ ext {
     }
     
     isRdaDataAvailable = false  // figure out how to set later
+    runSlowTests = false  // figure out how to set later
```

Comment:
So for the Travis or Jenkins category, we inspect the systems environmental variable to see if we are running in those environments as part of the gradle build. If so, then we set the gradle variables `isJenkins` and `isTravis` appropriately and let gradle exclude the appropriate categories. For example, if we're on Jenkins, skip anything in the category `ucar.unidata.util.test.category.NotJenkins`, and if we're on Travis skip anything in the categories `ucar.unidata.util.test.category.NotTravis`, or `ucar.unidata.util.test.category.NeedsExternalResource`. For this, we need to decide when we want to run the slow tests.

In this case, it might be best to allow runSlowTests to be set with a property we pass through gradle, say `./gradlew -DrunSlowTests=false testAll`, but we'll need to add some code to `gradle/root/testing.gradle` to look for that and set the variable properly.

Now, if you're running the test through intellij (directly through the junit run/debug option), the Category machinery in the gradle build will be ignored and the tests will just run, unless you explicitly tell junit to exclude categories by passing in `-DexcludedGroups=ucar.unidata.util.test.category.Slow` as a vm option...I think...